### ISSUE TITLE
Unable to make build locally 

### ISSUE THREAD

**Issue by camilamacedo86** [operator-framework/operator-registry#566]
The following error is faced when I try to build the project locally:

```
$ make build
GOFLAGS="-mod=vendor" go build  -tags "json1" -o bin/appregistry-server ./cmd/appregistry-server
GOFLAGS="-mod=vendor" go build  -tags "json1" -o bin/configmap-server ./cmd/configmap-server
GOFLAGS="-mod=vendor" go build  -tags "json1" -o bin/initializer ./cmd/initializer
# github.com/operator-framework/operator-registry/cmd/initializer
/usr/local/go/pkg/tool/darwin_amd64/link: running clang failed: exit status 1
ld: sectionForAddress(0xA347DE) address not in any section file '/var/folders/r_/ym6yn3cd0q3fm5jcgcwvj4z00000gn/T/go-link-543827867/go.o' for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

make: *** [Makefile:22: bin/initializer] Error 2
```

Should not it work? My env is go 1.15 and Mac OS

**Comment by ecordell** [comment]
I ran into this as well, there is an issue that is fixed in golang 1.15.5 (I think). I bumped to 1.15.6 and it builds fine now.

**Comment by benluddy** [comment]
Yes, this should be fixed starting in Go 1.15.4 (https://github.com/golang/go/issues?q=milestone%3AGo1.15.4+label%3ACherryPickApproved).
